bot/bot.py

```diff
@@ -290,11 +289,62 @@ async def error_handle(update: Update, context: CallbackContext) -> None:
     except:
         await context.bot.send_message(update.effective_chat.id, "Some error in error handler")
 
+
+async def error_handle_except_network(update: Update, context: CallbackContext) -> None:
+    logger.error(msg="Exception while handling an update:", exc_info=context.error)
+
+    if isinstance(context.error, NetworkError):
+        logger.debug(msg="ignore_network_error=True, supress network error")
+        return
+
+    try:
+        # collect error message
+        tb_list = traceback.format_exception(None, context.error, context.error.__traceback__)
+        tb_string = "".join(tb_list)[:2000]
+        update_str = update.to_dict() if isinstance(update, Update) else str(update)
+        message = (
+            f"An exception was raised while handling an update\n"
+            f"<pre>update = {html.escape(json.dumps(update_str, indent=2, ensure_ascii=False))}"
+            "</pre>\n\n"
+            f"<pre>{html.escape(tb_string)}</pre>"
+        )
+
+        # split text into multiple messages due to 4096 character limit
+        for message_chunk in split_text_into_chunks(message, 4000):
+            try:
+                await context.bot.send_message(update.effective_chat.id, message_chunk, parse_mode=ParseMode.HTML)
+            except telegram.error.BadRequest:
+                # answer has invalid characters, so we send it without parse_mode
+                await context.bot.send_message(update.effective_chat.id, message_chunk)
+            except NetworkError as e:
+                logger.error(msg="Exception while handling an update:", exc_info=e)
+    except:
+        await context.bot.send_message(update.effective_chat.id, "Some error in error handler")
+
+
 def run_bot() -> None:
+    global db
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-d", "--database", type=str)
+    parser.add_argument("-p", "--proxy", type=str)
+    curr_args = parser.parse_args()
+    if curr_args.database == "sqlite":
+        db = database_sqlite.SqliteDataBase(config.sqlite_database_uri)
+    else:
+        db = database_mongo.MongoDataBase(config.mongodb_uri)
```

Comment:
You would want to be able to run the bot with a docker-compose without starting a mongodb

Reply:
Sure, let me figure out how to modify the docker-compose file